src/azure-cli/azure/cli/command_modules/monitor/validators.py

```diff
@@ -317,3 +314,17 @@ def validate_action_group_ids(cmd, namespace):
             action_group_ids.append(group.lower())
         setattr(namespace, dest, action_group_ids)
     return validate_action_group_ids
+
+
+def validate_private_endpoint_connection_id(namespace):
+    if namespace.connection_id:
+        from azure.cli.core.util import parse_proxy_resource_id
+        result = parse_proxy_resource_id(namespace.connection_id)
+        namespace.resource_group_name = result['resource_group']
+        namespace.scope_name = result['name']
+        namespace.private_endpoint_connection_name = result['child_name_1']
+
+    if not all([namespace.scope_name, namespace.resource_group_name, namespace.private_endpoint_connection_name]):
+        raise CLIError('incorrect usage: [--id ID | --name NAME --scope-name NAME]')
```

Comment:
what does this error message tell user?

Reply:
choose one of these two. It's common error message shared within storage/keyvault/monitor

Reply:
add 'resource group name"?

---
In reply to: [407846027](https://github.com/Azure/azure-cli/pull/12931#discussion_r407846027) [](ancestors = 407846027)